### PR TITLE
Add trace_api logger to docs - 2.0

### LINKED ISSUE #8993: [docs] The listed loggers is incorrect
File: [docs/01_nodeos/06_logging/index.md](https://github.com/EOSIO/eos/blob/develop/docs/01_nodeos/06_logging/index.md)

The list of loggers is incorrect. The full list can be found here:
https://github.com/EOSIO/eos/blob/master/programs/nodeos/logging.json
with the addition of the newly added:  "trace_api"

### PR DESCRIPTION
**Pull request by lparisc**
## Change Description

Resolves #8993

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: docs/01_nodeos/06_logging/index.md [hunk 1]
```diff
@@ -105,4 +106,4 @@ Example:
 ```
 
 [[info]]
-| `net_plugin_impl`, `bnet_plugin`, `producer_plugin`, `transaction_tracing` are not enabled unless explicitly enabled in the `logging.json`
+| All loggers are enabled by default in the `logging.json`. All loggers have the `debug` logging level, except `net_plugin_impl` which is set to `info`.
```
Comment (heifner) [inline docs/01_nodeos/06_logging/index.md:109]: The default logging level for all loggers if no `logging.json` is provided is currently `info`. If you provide a `logging.json` file you can set each of loggers to whatever valid value you like.

File: docs/01_nodeos/06_logging/index.md [hunk 2]
```diff
@@ -105,4 +106,4 @@ Example:
 ```
 
 [[info]]
-| `net_plugin_impl`, `bnet_plugin`, `producer_plugin`, `transaction_tracing` are not enabled unless explicitly enabled in the `logging.json`
+| The default logging level for all loggers if no `logging.json` is provided is `info`. Each logger can be configured independently in the `logging.json` file.
```
Comment (iamveritas) [inline docs/01_nodeos/06_logging/index.md:109]: I would be nice if the doc will explicitly list the possible values for the logging level, right now it is clear two levels are possible: info and debug, but it is not clear if there are any others.

Comment (lparisc) [reply docs/01_nodeos/06_logging/index.md:109]: They're already listed [here](https://github.com/EOSIO/eos/blob/release/2.0.x/docs/01_nodeos/06_logging/01_logging-levels.md)

File: docs/01_nodeos/06_logging/index.md [hunk 3]
```diff
@@ -74,13 +74,14 @@ Example:
 
 ## Loggers
 
-The logging library built into EOSIO currently supports five loggers:
+The logging library built into EOSIO currently supports the following loggers:
 
 - `default` - the default logger, always enabled.
 - `net_plugin_impl` - detailed logging for the net plugin.
 - `bnet_plugin` - detailed logging for the bnet plugin.
```
Comment (heifner) [inline docs/01_nodeos/06_logging/index.md:81]: `bnet_plugin` is no longer valid. `bnet` was removed awhile ago.

Comment (heifner) [reply docs/01_nodeos/06_logging/index.md:81]: "`http_plugin` - detailed logging for the `http_plugin`."  is missing from the list.

Comment (lparisc) [reply docs/01_nodeos/06_logging/index.md:81]: Good catch - should've rechecked the list

**Pull request merged**